### DISCUSSION TITLE
Cursor blinking when out of focus

### DISCUSSION THREAD

**Discussion by tim-tim707** [WerWolv/ImHex#1526] [Feedback]
The cursor in the pattern editor blink even when ImHex is not focused. This doesn't stop to confuse me and I often begin to write to be surprise ImHex is not focused. I'm using windows and VSCode.

ImHex is really a high quality tool and I'm very impressed it is even free !

**Comment by paxcut** [comment]
Thank you for the feedback. I too have been victim of this problem and I found a couple of ways to check focus to imhex windows from the os point of view.  One doesn't seem to work but the other works good at least on windows. The fix is supposed to work on all glfw implementations but it needs to be tested to make sure. I'll add this change next time I create a pr and will let you know about it when it happens.

I just added a [pr](https://github.com/WerWolv/ImHex/pull/1528). Follow it to find out when/if it gets merged and to see the relevant changes to the source code.

**Comment by danielchasehooper** [comment]
Happens to me too. I'm on macOS and i notice it when a Sublime Text window is in front of the imhex window and the cursor is resting at a spot that overlaps both windows.

**Reply by paxcut** [reply]
I can't comment on Mac OS but on windows this was fixed a while back by a ImHex-wide focus detection that works with any other window from any app. Alt -tabbing also cycles focus with imhex windows as well. If it occurs for the pattern editor it must also occur for any other ImHex window that shows a cursor because pattern editor receives the focus from the app like all other windows.
